pkg/cri/server/container_start.go

```diff
@@ -64,8 +64,12 @@ func (c *criService) StartContainer(ctx context.Context, r *runtime.StartContain
 	}
 	defer func() {
 		if retErr != nil {
-			// Set container to exited if fail to start.
+			// Set container state depended on whether task is alive
```

Comment:
the original comment is not detailed enough... it not only sets the finished time indicating exited state.. it also sets errorStartExitCode which has further meaning:
```
	// errorStartExitCode is the exit code when fails to start container.
	// 128 is the same with Docker's behavior.
	// TODO(windows): Figure out what should be used for windows.
	errorStartExitCode = 128
```

thus.. I wonder if this change is needed